eng/common/scripts/Test-Spelling.ps1

```diff
@@ -0,0 +1,87 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $TargetRef = 'master', 
+
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $CspellConfigPath = "./.vscode/cspell.json"
```

Comment:
Does anything bad happen here if this file doesn't exist?

Reply:
cspell will exit with an error if there is no config. To that end we could validate presence of the file and exit early if it's not there. 

```
> cspell --config doesnotexist.json **/*
Configuration Error: Failed to resolve file: "doesnotexist.json"
CSpell: Files checked: 0, Issues found: 0 in 0 files
```